app/src/test/java/com/severianfw/picto/usecase/GetPhotoUseCaseTest.kt

```diff
@@ -0,0 +1,94 @@
+package com.severianfw.picto.usecase
+
+import com.severianfw.picto.connectionlistener.InternetConnectionListener
+import com.severianfw.picto.data.remote.PhotoResponse
+import com.severianfw.picto.data.repository.PhotoRepository
+import com.severianfw.picto.domain.model.PhotoItemModel
+import com.severianfw.picto.domain.usecase.impl.GetPhotoUseCaseImpl
+import com.severianfw.picto.utils.PhotoMapper
+import io.reactivex.rxjava3.core.Single
+import org.junit.After
+import org.junit.Assert
+import org.junit.Before
+import org.junit.Test
+import org.mockito.InjectMocks
+import org.mockito.Mock
+import org.mockito.Mockito
+import org.mockito.Mockito.doNothing
```

Comment:
Let's be consistent. Either `Mockito.when` or `when` , `Mockito.doNothing` or `doNothing`